---
title: "23 May Markdow"
author: "RLC"
date: "May 23, 2016"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

## R Markdown

This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.

When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:

```{r cars}
summary(cars)
```

## Including Plots

You can also embed plots, for example:

```{r pressure, echo=FALSE}
plot(pressure)
```

Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.


```{r}
file <- "gencode.v24.annotation.gff3"
cat("Loading", file, "at", date(), " ... \n")
GFF <- read.table(file, header=F, sep="\t", nrows=1000)
colnames(GFF) <- c("seqid", "source", "type", "start", "end", "score", "strand", "phase", "attributes")
rownames(GFF) <- 1:nrow(GFF)
```


```{r  Install of Biostrings package}
 if(!require("Biostrings")){
   source("http://bioconductor.org/biocLite.R")
   biocLite("Biostrings")
   require("Biostrings")
   }
```

```{r}
print(date())
genome <- readDNAStringSet("Arabidopsis_thaliana.TAIR10.31.dna.toplevel.fa") 
names(genome)
seq <- subseq(genome[[3]], start=10000, end=10009) 
print(seq)
writeXStringSet(DNAStringSet(seq), file="seq.fasta")
print(date())
```

